### PR TITLE
feat(ContentSwitcher): add experimental ContentSwitcher

### PR DESCRIPTION
**Pull request by emyarod**
Closes IBM/carbon-components-react#1715

This PR will match the experimental ContentSwitcher to its vanilla counterpart

depends on https://github.com/IBM/carbon-components/pull/1603

### CODE REVIEW

**Comment by asudoh** [comment]
@emyarod GIven IBM/carbon-components#1603 has been merged, can we go forward with this...? Thanks!

**Comment by asudoh** [comment]
Thanks @emyarod! Is it possible for you to take a look at the latest CI status...?

**Comment by emyarod** [comment]
@asudoh it should be resolved now, just need to increase the bundle size threshold some more

File: src/components/Switch/Switch.js [hunk 1]
```diff
@@ -56,15 +56,15 @@ const Switch = props => {
     return (
       <button {...other} {...commonProps}>
         {btnIcon}
-        {text}
+        <span>{text}</span>
```
Comment (joshblack) [inline src/components/Switch/Switch.js]: Is this part of the changes for X?

Comment (emyarod) [reply src/components/Switch/Switch.js]: yes it is meant to match [this line in vanilla](https://github.com/IBM/carbon-components/blob/master/src/components/content-switcher/content-switcher.hbs#L12)

**Pull request merged**